package.json

```diff
@@ -11,6 +11,11 @@
   "author": "pholyoke@kenzan.com",
   "license": "MIT",
   "devDependencies": {
-    "eslint": "^4.8.0"
+    "archiver": "^2.1.0",
+    "eslint": "^4.8.0",
+    "rimraf": "^2.6.2"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.154.0"
```

Comment:
not a big deal, but I like to order `dependencies` before `devDependencies`

Reply:
Not sure why this happened. I never manually edited it to be this way, I'll fix it though.